src/lib/mongoBackend/mongoSubscribeContext.cpp

```diff
@@ -202,7 +203,7 @@ HttpStatusCode mongoSubscribeContext
     }
 
     /* Adding format to use in notifications */
-    sub.append(CSUB_FORMAT, "JSON");
+    sub.append(CSUB_FORMAT, notificationFormatToString(requestP->attrsFormat));
```

Comment:
requestP->attrsFormat are filled in during the parse step, and set to default values before the parse step